src/common/utils.ts

```diff
@@ -493,9 +493,9 @@ const fetchLoadBalancerIngressData = async (): Promise<string> => {
       await $`kubectl get -n ingress svc nginx-ingress-controller -o jsonpath="{.status.loadBalancer.ingress}"`
     ).stdout.trim()
     count += 1
-    if (isEmpty(ingressDataString)) break
+    if (!ingressDataString) break
```

Comment:
Why has this changed?  ```[{"ip":"x.x.x.x"}]``` input is expected here right, not a boolean?

Reply:
```suggestion
    if (ingressDataString === '') break
```

Reply:
Please stick to design by contract. If it only returns strings, then we should compare to strings. If it can be `undefined` we check that with `=== undefined`.